### ISSUE TITLE
Remove filter buttons lacking role attribute

### ISSUE THREAD

**Issue by Valar05** [JedWatson/react-select#5059]
My company is currently doing accessibility reviews of various front-end code.  We've come across an issue where the remove filter buttons on the multiselect lack the role attribute.  Would it be possible for this to be added?  It would be for the individual filters as well as the remove all filters button.

**Comment by all141** [comment]
Hello @Methuselah96,

I'd like to take this on as a starter issue to get used to the project. Contributing guide told me to let someone know if I'd like to work on a fix. Could this please be assigned to me?

Thanks!

**Comment by all141** [comment]
I might have tagged the wrong person. @gwyneplaine and or @JedWatson:

I'd like to take this on as a starter issue to get used to the project. Contributing guide told me to let someone know if I'd like to work on a fix. Could this please be assigned to me? I have a PR ready.

Thanks!

**Comment by Rall3n** [comment]
Hello @all141,

you are not required to be assigned an issue to create a PR for it. Mentioning the issue in your PR should be sufficient.

If you still insist, then I can assign you this issue.
